### PR TITLE
add homework lesson - 5 level - 3 / Task 16-24

### PR DESCRIPTION
**Pull request by Raivuha**

### CODE REVIEW

**Review by Raivuha — COMMENTED** [review]
Sorry for inccorect commit.

File: src/main/java/students/student_raivis_urbevics/lesson5_level3/task17/ForLoopDefinition2.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package students.student_raivis_urbevics.lesson5_level3.task17;
+
+        class ForLoopDefinition2 {
+
+         public static void main(String[] args) {
+        for (int i = 0; i < 10; i++) {
```
Comment (antdolg1) [inline src/main/java/students/student_raivis_urbevics/lesson5_level3/task17/ForLoopDefinition2.java:6]: Missing formatting.

File: src/main/java/students/student_raivis_urbevics/lesson5_level3/task18/ForLoopDefinition3.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package students.student_raivis_urbevics.lesson5_level3.task18;
+
+             class ForLoopDefinition3 {
+
+               public static void main(String[] args){
+                  for (int i = 0; i < 10 ; i++) {
+            System.out.println(i);
+        }
+    }
+}
```
Comment (antdolg1) [inline src/main/java/students/student_raivis_urbevics/lesson5_level3/task18/ForLoopDefinition3.java:10]: Formatting missing.

**Review by antdolg1 — APPROVED** [review]
All good but you are missing proper formatting in all your classes. Please further don't forget to format your code.

**Pull request merged**